package.json

```diff
@@ -116,43 +111,32 @@
 		"core-js": "3.6.5",
 		"create-file-webpack": "1.0.2",
 		"cross-env": "6.0.3",
-		"css-loader": "3.6.0",
```

Comment:
I'm not sure to understand why some dependencies were removed.

Reply:
They are now included in the version of `wp-scripts` we're shipping with. It makes it easier for managing these dependencies because we just need to update `wp-scripts`.

Reply:
I should add, there may be more dependencies we can remove as well, I didn't want to get too carried away in this pull though.